modules/m_who.c

```diff
@@ -171,7 +171,8 @@ m_who(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source_p,
 		return;
 	}
 
-	if(IsOperSpy(source_p) && *mask == '!')
+	if(*mask == '!'
+		&& HasPrivilege(source_p, IsChannelName(mask + 1) ? "oper:spy_channel" : "oper:spy_user"))
```

Comment:
```suggestion
		&& (IsChannelName(mask + 1) ? IsOperSpyChannel(source_p) : IsOperSpyUser(source_p)))
```

Reply:
that's what this was going to be, but @edk0 expressed a preference for moving away from those macros